airsonic-main/src/main/webapp/WEB-INF/jsp/generalSettings.jsp

```diff
@@ -80,6 +80,18 @@
 
         <tr><td colspan="2">&nbsp;</td></tr>
 
+        <tr>
+            <td><fmt:message key="generalsettings.adminuser"/></td>
+            <td>
+                <form:select path="adminUser" cssStyle="width:15em">
+                    <c:forEach items="${command.usernames}" var="username">
+                        <form:option value="${username}" label="${username}"/>
```

Comment:
Please escape the values with `fn:escapeXml` otherwise we'll have XSS issues here.

Reply:
Done